### ISSUE TITLE
[Patients] Menu elements missing role

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#6406]
## Area or Component

Patients page -> menus: 
1- Add patients
2- More actions

## Quick Summary

There are two menus in the patients page that display actionable functionality to the user. These two elements are currently mark as a button instead of a [menu](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menu_role).

### How to Replicate

Go to the patients page and verify the two elements are missing the role="menu"

## Accessibility Violation

[The element's role is missing or incorrect](https://docs.deque.com/issue-help/1.0.0/en/aria-role-missing).

## Action Requested

Add `role="menu"` to both elements.

## Screenshots 
![Screenshot 2023-08-23 at 11 56 31 AM](https://github.com/CDCgov/prime-simplereport/assets/103958711/220f1ce5-e8d8-4f1b-afce-ff4b993d6d0e)

**Comment by johanna-skylight** [comment]
Closing after manually testing with voice over and realizing this is a false positive.
